src/tests/suite/daos_test.c

```diff
@@ -162,6 +163,13 @@ run_specified_tests(const char *tests, int rank, int size,
 			nr_failed += run_daos_checksum_test(rank, size,
 						sub_tests, sub_tests_size);
 			break;
+		case 'Z':
+			daos_test_print(rank, "\n\n=================");
+			daos_test_print(rank, "DAOS EC aggregation tests..");
+			daos_test_print(rank, "=================");
+			nr_failed += run_daos_aggregation_ec_test(rank, size,
+						sub_tests, sub_tests_size);
+			break;
```

Comment:
if this works, could you please add the test to src/tests/ftest/daos_test/daos_core_test.yaml ,  Thanks.